### ISSUE TITLE
Assign roles to users automatically.

### ISSUE THREAD

**Issue by Tonypsilon** [Tonypsilon/bmm.backend#14]
As an administrator of a club, a season or the whole application, I want to be able to assign certain roles to users. If I make someone a clubadmin for a club (as an example), I expect that the respective user gets the CLUB_ADMIN role assigned in case he is not a CLUB_ADMIN yet (which is currently not the case). It should happen implicitly.  
The revocation of roles is not part of this story.

**Comment by Tonypsilon** [comment]
Once this is done, during user creation no roles must be provided to a user.
